GVFS/GVFS.UnitTests/Virtualization/Projection/SortedFolderEntriesTests.cs

```diff
@@ -117,15 +117,45 @@ public void ValidateOrderOfDefaultEntries()
         }
 
         [TestCase]
-        public void AddItemWithIsIncludedFalse()
+        public void AddFolderNoIncludeChildrenShouldBeIncluded()
         {
             SortedFolderEntries sfe = SetupDefaultEntries();
             LazyUTF8String name = ConstructLazyUTF8String("IsIncludedFalse");
-            sfe.AddFile(name, new byte[20], isIncluded: false);
-            sfe.TryGetValue(name, out FolderEntryData folderEntryData).ShouldBeTrue();
-            folderEntryData.ShouldNotBeNull();
-            folderEntryData.IsFolder.ShouldBeFalse();
-            folderEntryData.IsIncluded.ShouldBeFalse();
+            sfe.GetOrAddFolder(new[] { name }, partIndex: 0, parentIsIncluded: false, rootSparseFolderData: new SparseFolderData());
+            ValidateFolder(sfe, name, isIncludedValue: true);

```

Comment:
Is `isIncludedValue` true because `rootSparseFolderData` is empty?

I had some trouble parsing the name of this test, perhaps something like `AddFolderWhenSparseFolderDataIsEmtpy` or `FoldersShouldBeVisibleWhenSparseFolderDataIsEmpty`?

Reply:
>Is isIncludedValue true because rootSparseFolderData is empty?

Yes because if the root sparse folder data doesn't have any children then everything should be included.

I will update the name of the test.